app/script/tracking/EventTrackingRepository.coffee

```diff
@@ -246,6 +247,25 @@ class z.tracking.EventTrackingRepository
     Raygun.setVersion z.util.Environment.version false if not z.util.Environment.frontend.is_localhost()
     Raygun.withCustomData {electron_version: z.util.Environment.version true} if z.util.Environment.electron
     Raygun.onBeforeSend @_check_error_payload
+    @_attach_unhandled_rejected_promises()
+
+  # Attach to rejected Promises.
+  _attach_unhandled_rejected_promises: ->
+    window.onunhandledrejection = (event) =>
```

Comment:
Let's test this behaviour with a Promise which immediately rejects in our test suite.